### ISSUE TITLE
Some introspection about molecular structure

### ISSUE THREAD

**Issue by Yurlungur** [lanl/singularity-eos#442]
In his review of #439, @pdmullen pointed out that it would be nice to initialize an ideal gas with, e.g., 
```
IdealGas(Real gm1, MeanAtomicProperties prop)
```
and compute Cv automatically from that. But this would require knowledge of, e.g., the mean molecular weight $\mu$, rather than the mean atomic mass.

This may also be relevant for more detailed 3T models. We should consider a strategy to enable this down the line. Perhaps, for example, by storing the number of atoms in a molecule in the `MeanAtomicProperties` struct, or by passing this in to the ideal gas constructor.

**Comment by jhp-lanl** [comment]
3T usually implies ionization and ionization (at least for gasses) implies dissociated atoms rather than molecules. In other words I'd think that for most 3T applications, you'd choose the atomic form of the species of interest.

If we get to an ionization model, we might want to differentiate between mean atomic mass and mean molecular mass so that an ionized molecule can dissociate.

**Comment by Yurlungur** [comment]
Agreed with this assessment. I think for now this implies the introspection for molecules is not relevant. Closing this issue for now.

**Comment by Yurlungur** [comment]
This came up again---perhaps for a non-molecular ideal gas, we should have the ability to construct $C_v$ from $\Gamma$ and mean atomic properties.
